### PR TITLE
Add PledgeIt to companies section

### PR DESCRIPTION
**Pull request by joshlasdin**
Added Pledge It to the companies section of the homepage. Our SPA is now all marionette :smile:

### CODE REVIEW

File: src/sections/_companies.jade [hunk 1]
```diff
@@ -202,4 +202,7 @@ section.logos.column-contain.hidden-xs
       li
         a(href= "https://www.zweitag.de", target="_blank")
           img(data-src="images/logos/zweitag.png", alt= "zweitag")
+      li
+        a(href= "https://pledgeit.org", target="_blank")
```
Comment (samccone) [inline src/sections/_companies.jade]: alphabetical plz :)

Comment (joshlasdin) [inline src/sections/_companies.jade]: Fixed in e599ba2. BeSnappy threw me off :P

**Comment by samccone** [comment]
mind smashing this down into 1 commit?

**Comment by joshlasdin** [comment]
Not at all

**Comment by jdaudier** [comment]
@joshlasdin This looks like one commit. I guess you did a force push? Trying to figure out if I should merge it.

**Comment by samccone** [comment]
👍👍👍

**Comment by jdaudier** [comment]
thanks @samccone!  sorry that took forever @joshlasdin

**Comment by joshlasdin** [comment]
Rad! Thanks guys!!

**Comment by samccone** [comment]
Yeah sorry about the delay! my bad :(

**Comment by joshlasdin** [comment]
@samccone No problem, I know things get busy :)

**Pull request merged**